### PR TITLE
Maven artifact fix

### PR DESCRIPTION
**Pull request by ahartwellCpointe**

### CODE REVIEW

File: src/test/java/io/kokuwa/maven/helm/PackageMojoTest.java [hunk 1]
```diff
@@ -40,20 +45,26 @@ void dependencies(PackageMojo mojo) {
...
+				"1.0.0", "", "", "", null));
 		assertHelm(mojo, "package src/test/resources/simple --destination target/helm/repo --version 0.0.1");
 	}
 
 	@DisplayName("with chartVersion snapshot")
 	@Test
 	void chartVersionSnapshot(PackageMojo mojo) {
 		mojo.setChartVersion("0.0.1-SNAPSHOT");
+		mojo.mavenProject.setArtifact(new DefaultArtifact("io.kokuwa.maven", "helm-maven-plugin",
+				"1.0.0", "", "", "", null));
```
Comment (csun-cpointe) [inline src/test/java/io/kokuwa/maven/helm/PackageMojoTest.java:67]: Q: not quite understand the purpose of setting this artifact for the test

File: src/main/java/io/kokuwa/maven/helm/PackageMojo.java [hunk 2]
```diff
@@ -116,9 +126,28 @@ public void execute() throws MojoExecutionException {
 			}
 
 			helm.execute("Unable to package chart at " + chartDirectory);
+			setUpPlaceholderFileAsMavenArtifact();
 		}
 	}
 
+	protected void setUpPlaceholderFileAsMavenArtifact() {
```
Comment (csun-cpointe) [inline src/main/java/io/kokuwa/maven/helm/PackageMojo.java:133]: A: please add a comment to describe the purpose the function

**Review by csun-cpointe — APPROVED** [review]

File: src/main/java/io/kokuwa/maven/helm/PackageMojo.java (hunk 2 above)
Comment (ahartwellCpointe) [reply src/main/java/io/kokuwa/maven/helm/PackageMojo.java:133]: Done

File: src/test/java/io/kokuwa/maven/helm/PackageMojoTest.java (hunk 1 above)
Comment (ahartwellCpointe) [reply src/test/java/io/kokuwa/maven/helm/PackageMojoTest.java:67]: The test project that it is using here does not have an artifact, which will cause a null pointer in the package mojo when we try to assign our dummy file to an artifact that doesn't exist. So, I'm just setting one programmatically so the tests can proceed

**Review by Cho-William — APPROVED** [review]

**Pull request merged**